packages/catalog-search/src/SearchSuggestions.jsx

```diff
@@ -52,6 +56,28 @@ const SearchSuggestions = ({
             ))
         }
       </div>
+      )}
+      {/* Currently (Feb 2023) it is not possible to redirect to the learner portal for exec ed content so only display
+       if the redirect is disabled */}
+      {execEdCourses.length > 0 && disableSuggestionRedirect && (
+      <div>
+        <div className="mb-2 mt-5 ml-2 font-weight-bold suggestions-section">
+          Executive Education
+        </div>
+        {
+          execEdCourses.slice(0, MAX_NUM_SUGGESTIONS)
+            .map((hit) => (
+              <SearchSuggestionItem
+                key={hit.title}
+                url={getLinkToCourse(hit)}
```

Comment:
[inform] Through https://2u-internal.atlassian.net/browse/ENT-6966, in order to support the co-branded header with the GetSmarter logo on learner portal course pages (i.e., when learner is viewing an Exec Ed course, show both customer logo & GetSmarter logo side by side), we've opted to create a new route to handle non-OCM course types, e.g.: `https://enterprise.edx.org/<slug>/executive-education-2u/course/<course_key>` which means `getLinkToCourse` ([source](https://github.com/openedx/frontend-enterprise/blob/b31a1ff73b27ad5047d1bb4a22d228498159709b/packages/catalog-search/src/SearchSuggestions.jsx#L14)) will likely need to support course types. The course pages are otherwise identical except for the co-branded header.

We do have logic in place where if the learner lands on the OCM course page route for an Exec Ed course, we redirect them to the route with the `courseType` in it so the logo appears.

[context] The rationale for this decision about course type specific course about pages is that we want to show the co-branded logos with GetSmarter immediately on page load so that the GetSmarter logo doesn't just randomly appear in the header after the course metadata from course-discovery API finishes loading. This is a shortcoming in our MFE architecture, where it'd be _really_ nice if we could fetch the prerequisite data needed for an MFE route on the server instead of at runtime, after the entire JS bundle gets loaded/processed by the browser.